### PR TITLE
task: [Sign & Submit] Display field-level alerts as Errors instead of Warnings

### LINKED ISSUE #1053: [Sign & Submit] Verify alerts for missing Institution info
- [x] LEI
- [x] TIN
- [x] Types of Financial Institution

### PR DESCRIPTION
**Pull request by meissadia**
Closes #1053 

Sorry, this turned into a stir fry with lots of language updates mixed in.

## Changes

- [Sign and Submit] Display missing LEI, TIN, or Type of FI as errors
- task: Address linting errors associated with empty Institution `data` when API calls fail
- fix: Language updates which have been verified by Natalia [here](https://github.com/cfpb/sbl-frontend/pull/1052#issuecomment-2484213497)
- fix: More language updates which have been verified by Natalia [here](https://github.com/cfpb/sbl-frontend/pull/1052#issuecomment-2486690884)

## Screenshots

- [Sign and Submit] Display missing LEI, TIN, or Type of FI as errors
    - screencapture-localhost-8899-filing-2024-123456789TESTBANK123-submit-2024-11-18-11_43_49

### CODE REVIEW

**Review by natalia-fitzgerald — DISMISSED** [review]
Looks great!

File: src/pages/Filing/ViewInstitutionProfile/DisplayField.tsx [hunk 1]
```diff
@@ -49,27 +50,29 @@ export interface DisplayFieldProperties {
...
         </Heading>
       ) : undefined}
-      <p className='u-mt10 break-all'>{resultingValue}</p>
+      <p className='u-mt10'>{resultingValue}</p>
       <AlertFieldLevel
-        status='warning'
+        status={alertStatus}
         isVisible={showAlert}
         message={NotProvidedAlertMessage[label as string]}
       />
```
Comment (meissadia) [inline src/pages/Filing/ViewInstitutionProfile/DisplayField.tsx:78]: Allow field-level alerts to be displayed as either `error` or `warning`

File: src/pages/Filing/FilingApp/FilingSubmit.tsx [hunk 2]
```diff
@@ -194,6 +195,7 @@ export function FilingSubmit(): JSX.Element {
               heading='Confirm your financial institution identifying information'
               data={institution}
               description={getDescriptionForSignAndSubmitSection()}
+              alertStatus='error'
```
Comment (meissadia) [inline src/pages/Filing/FilingApp/FilingSubmit.tsx:198]: Show `Sign and Submit` page alerts as "errors"

File: src/pages/Filing/UpdateFinancialProfile/UfpForm.tsx [hunk 3]
```diff
@@ -34,16 +34,18 @@ export default function UFPForm({
...
 }: {
-  data: InstitutionDetailsApiType;
+  data: InstitutionDetailsApiType | undefined;
   isError: boolean;
 }): JSXElement {
   const { lei } = useParams();
   const isRoutingEnabled = getIsRoutingEnabled();
   const navigate = useNavigate();
 
+  const dataIsMissing = isError || !data;
```
Comment (meissadia) [inline src/pages/Filing/UpdateFinancialProfile/UfpForm.tsx:44]: All of these checks for `!data` are to appease the linter

**Comment by meissadia** [comment]
Merge conflicts resolved.

**Review by tanner-ricks — APPROVED** [review]

**Pull request merged**